Eatery/Views/Eateries/CampusMenuInfoView.swift

```diff
@@ -54,14 +67,15 @@ class CampusMenuInfoView: UIView, MenuInfoView {
             make.leading.equalTo(statusLabel.snp.trailing).offset(2)
         }
 
-        locationLabel.isOpaque = false
-        locationLabel.font = UIFont.systemFont(ofSize: 14, weight: .medium)
-        locationLabel.textColor = .gray
-        addSubview(locationLabel)
-        locationLabel.snp.makeConstraints { make in
-            make.leading.equalToSuperview().offset(16)
-            make.top.equalTo(statusLabel.snp.bottom).offset(8)
-            make.bottom.equalToSuperview().inset(16)
+        moreHoursButton.setImage(UIImage(named: "upArrow")?.withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
I don't think you need to say `.withRenderingMode(.alwaysTemplate)`. Instead, you can set it to `Render As [Template Image]` in the icon's file in the assets folder.

Reply:
Eatery uses the up arrow image elsewhere and I didn't want to influence that